Dockerfile

```
@@ -0,0 +1,32 @@
+FROM ubuntu:22.04 as build
+ENV TZ=UTC
+
+LABEL Cammoguy <jpburnett@gwu.edu>
+
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone && \
+    apt-get update && apt-get install -y \
+    build-essential \
+    binutils-mips-linux-gnu \
+    pkg-config \
+    python3 \
+    python3-pip \
+    git \
+    wget \
+    unzip \
+    clang-tidy-11 \
+    clang-format-11 \
+    libpng-dev && \
+    apt clean && \
+    rm -rf /var/lib/apt/lists/*
+
+COPY requirements.txt requirements.txt
+
+RUN python3 -m pip install --no-cache-dir -r requirements.txt
```

```suggestion
RUN python3 -m pip install --no-cache-dir colorama watchdog python-Levenshtein cxxfilt \
                                              pycparser -r requirements.txt
```
Requested change: more dependencies installed for tools

These were required to get mips2c and diff.py working.
Formating might be weird, no preference if you want to change it.

---

PR #918 adds those to the `requirements.txt` file, so if that one gets in there should be no need to add those explicitly here